### DISCUSSION TITLE
Populate App Resources with common, global, and discipline views by default

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5830] [Feature Requests]
<img width="831" alt="image" src="https://github.com/specify/specify7/assets/37256050/4a59b417-5c0b-4af1-9847-61f8ed9d1472">

See here:

https://discourse.specifysoftware.org/t/editing-forms/736#default-views-2

You cannot really start form editing in Specify 7 without either a custom view or downloading the 6 files and editing them, then creating resources in the application.

By default, these three types need to be exposed to the user in the application. They can then pull from global and common views and make their own view definitions or modify the defaults.

The discipline-specific views should be under each discipline. It would be so much more intuitive, as Specify 7 already uses these files from the static Specify 6 files, it just doesn't show them in the UI.

@maxpatiiuk Do you understand what I mean?

**Comment by melton-jason** [comment]
There is already a feature very similar to this on the `xml-editor` branch. 
When creating a new Form Definition, Specify lists the defaults and allows you to choose which one to use.


https://github.com/specify/specify7/assets/64045831/92bc8ca7-db04-4904-a395-54f97c5f965b

Though I admit it would be helpful to display the default views from Specify 6 the application is already using in the App Resources.

**Comment by maxpatiiuk** [comment]
@grantfitzsimmons I belive this is fixed on the xml-editor branch. Can you check if there is anything there that's not yet implemented?

**Comment by grantfitzsimmons** [comment]
@maxpatiiuk @melton-jason I'm not sold– yet. These need to exist _already_ to make it easier for users currently, but once we make the UI intuitive for users on `xml-editor`, maybe not!
